.github/macos-build.sh

```diff
@@ -48,8 +48,7 @@ mv src/.libs.combined src/.libs
 sudo make install
 cd ../..
 
-python -m pip install --upgrade setuptools wheel pip setuptools-scm
+python -m pip install --upgrade pip
 python -m pip install cibuildwheel==2.21.3
-pip install -r requirements.txt
```

Comment:
There is no need to install the removed packages as they will be automatically installed in an isolated environment during the build.